### ISSUE TITLE
Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61

### ISSUE THREAD

**Issue by teracube** [CortexPE/TeaSpoon#88]
## Issue Description

Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61

## Steps to Reproduce

1. ...PocketMine-MP version 1.7dev-525 "[REDACTED]" (API 3.0.0-ALPHA10)
2. ...TeaSpoon v1.0.0 =) TeaSpoon_dev-51.phar
3. ...

## Crashdump, Backtrace or Other Files

```2017-12-20 [18:20:11] [Server thread/CRITICAL]: Could not tick level "lobby": Call to undefined method CortexPE\Server::getInstance()
2017-12-20 [18:20:11] [Server thread/CRITICAL]: Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61
2017-12-20 [18:20:11] [Server thread/DEBUG]: #0 src/pocketmine/Player(1449): CortexPE\block\Lava->onEntityCollide(pocketmine\Player object)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #1 src/pocketmine/entity/Entity(1595): pocketmine\Player->checkBlockCollision()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #2 src/pocketmine/Player(1502): pocketmine\entity\Entity->move(double -0.057000000000016, double 0, double -0.31960000000004)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #3 src/pocketmine/Player(1662): pocketmine\Player->processMovement(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #4 src/pocketmine/level/Level(751): pocketmine\Player->onUpdate(integer 151969)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #5 src/pocketmine/Server(2377): pocketmine\level\Level->doTick(integer 151969)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #6 src/pocketmine/Server(2534): pocketmine\Server->checkTickUpdates(integer 151969, double 1513790411.8109)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #7 src/pocketmine/Server(2274): pocketmine\Server->tick()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #8 src/pocketmine/Server(2149): pocketmine\Server->tickProcessor()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #9 src/pocketmine/Server(1737): pocketmine\Server->start()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #10 src/pocketmine/PocketMine(554): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g3.hmserv.eu-19144/, string /home/gsp_1485/g3.hmserv.eu-19144/plugins/)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #11 /home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2017-12-20 [18:20:11] [Server thread/CRITICAL]: Could not tick level "lobby": Call to undefined method CortexPE\Server::getInstance()
2017-12-20 [18:20:11] [Server thread/CRITICAL]: Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61
2017-12-20 [18:20:11] [Server thread/DEBUG]: #0 src/pocketmine/Player(1449): CortexPE\block\Lava->onEntityCollide(pocketmine\Player object)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #1 src/pocketmine/entity/Entity(961): pocketmine\Player->checkBlockCollision()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #2 src/pocketmine/entity/Living(513): pocketmine\entity\Entity->entityBaseTick(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #3 src/pocketmine/entity/Human(403): pocketmine\entity\Living->entityBaseTick(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #4 src/pocketmine/Player(1665): pocketmine\entity\Human->entityBaseTick(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #5 src/pocketmine/level/Level(751): pocketmine\Player->onUpdate(integer 151970)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #6 src/pocketmine/Server(2377): pocketmine\level\Level->doTick(integer 151970)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #7 src/pocketmine/Server(2534): pocketmine\Server->checkTickUpdates(integer 151970, double 1513790411.8609)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #8 src/pocketmine/Server(2274): pocketmine\Server->tick()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #9 src/pocketmine/Server(2149): pocketmine\Server->tickProcessor()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #10 src/pocketmine/Server(1737): pocketmine\Server->start()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #11 src/pocketmine/PocketMine(554): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g3.hmserv.eu-19144/, string /home/gsp_1485/g3.hmserv.eu-19144/plugins/)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #12 /home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2017-12-20 [18:20:11] [Server thread/CRITICAL]: Could not tick level "lobby": Call to undefined method CortexPE\Server::getInstance()
2017-12-20 [18:20:11] [Server thread/CRITICAL]: Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61
2017-12-20 [18:20:11] [Server thread/DEBUG]: #0 src/pocketmine/Player(1449): CortexPE\block\Lava->onEntityCollide(pocketmine\Player object)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #1 src/pocketmine/entity/Entity(961): pocketmine\Player->checkBlockCollision()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #2 src/pocketmine/entity/Living(513): pocketmine\entity\Entity->entityBaseTick(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #3 src/pocketmine/entity/Human(403): pocketmine\entity\Living->entityBaseTick(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #4 src/pocketmine/Player(1665): pocketmine\entity\Human->entityBaseTick(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #5 src/pocketmine/level/Level(751): pocketmine\Player->onUpdate(integer 151971)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #6 src/pocketmine/Server(2377): pocketmine\level\Level->doTick(integer 151971)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #7 src/pocketmine/Server(2534): pocketmine\Server->checkTickUpdates(integer 151971, double 1513790411.9109)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #8 src/pocketmine/Server(2274): pocketmine\Server->tick()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #9 src/pocketmine/Server(2149): pocketmine\Server->tickProcessor()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #10 src/pocketmine/Server(1737): pocketmine\Server->start()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #11 src/pocketmine/PocketMine(554): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g3.hmserv.eu-19144/, string /home/gsp_1485/g3.hmserv.eu-19144/plugins/)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #12 /home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2017-12-20 [18:20:11] [Server thread/CRITICAL]: Could not tick level "lobby": Call to undefined method CortexPE\Server::getInstance()
2017-12-20 [18:20:11] [Server thread/CRITICAL]: Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61
2017-12-20 [18:20:11] [Server thread/DEBUG]: #0 src/pocketmine/Player(1449): CortexPE\block\Lava->onEntityCollide(pocketmine\Player object)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #1 src/pocketmine/entity/Entity(1595): pocketmine\Player->checkBlockCollision()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #2 src/pocketmine/Player(1502): pocketmine\entity\Entity->move(double -0.02849999999998, double 0, double -0.16039999999998)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #3 src/pocketmine/Player(1662): pocketmine\Player->processMovement(integer 1)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #4 src/pocketmine/level/Level(751): pocketmine\Player->onUpdate(integer 151972)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #5 src/pocketmine/Server(2377): pocketmine\level\Level->doTick(integer 151972)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #6 src/pocketmine/Server(2534): pocketmine\Server->checkTickUpdates(integer 151972, double 1513790411.9609)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #7 src/pocketmine/Server(2274): pocketmine\Server->tick()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #8 src/pocketmine/Server(2149): pocketmine\Server->tickProcessor()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #9 src/pocketmine/Server(1737): pocketmine\Server->start()
2017-12-20 [18:20:11] [Server thread/DEBUG]: #10 src/pocketmine/PocketMine(554): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g3.hmserv.eu-19144/, string /home/gsp_1485/g3.hmserv.eu-19144/plugins/)
2017-12-20 [18:20:11] [Server thread/DEBUG]: #11 /home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2017-12-20 [18:20:12] [Server thread/CRITICAL]: Could not tick level "lobby": Call to undefined method CortexPE\Server::getInstance()
2017-12-20 [18:20:12] [Server thread/CRITICAL]: Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61
2017-12-20 [18:20:12] [Server thread/DEBUG]: #0 src/pocketmine/Player(1449): CortexPE\block\Lava->onEntityCollide(pocketmine\Player object)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #1 src/pocketmine/entity/Entity(1595): pocketmine\Player->checkBlockCollision()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #2 src/pocketmine/Player(1502): pocketmine\entity\Entity->move(double -0.081000000000017, double 0.105, double -0.45479999999998)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #3 src/pocketmine/Player(1662): pocketmine\Player->processMovement(integer 1)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #4 src/pocketmine/level/Level(751): pocketmine\Player->onUpdate(integer 151973)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #5 src/pocketmine/Server(2377): pocketmine\level\Level->doTick(integer 151973)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #6 src/pocketmine/Server(2534): pocketmine\Server->checkTickUpdates(integer 151973, double 1513790412.0109)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #7 src/pocketmine/Server(2274): pocketmine\Server->tick()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #8 src/pocketmine/Server(2149): pocketmine\Server->tickProcessor()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #9 src/pocketmine/Server(1737): pocketmine\Server->start()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #10 src/pocketmine/PocketMine(554): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g3.hmserv.eu-19144/, string /home/gsp_1485/g3.hmserv.eu-19144/plugins/)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #11 /home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2017-12-20 [18:20:12] [Server thread/CRITICAL]: Could not tick level "lobby": Call to undefined method CortexPE\Server::getInstance()
2017-12-20 [18:20:12] [Server thread/CRITICAL]: Error: "Call to undefined method CortexPE\Server::getInstance()" (EXCEPTION) in "TeaSpoon_dev-51.phar/src/CortexPE/block/Lava" at line 61
2017-12-20 [18:20:12] [Server thread/DEBUG]: #0 src/pocketmine/Player(1449): CortexPE\block\Lava->onEntityCollide(pocketmine\Player object)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #1 src/pocketmine/entity/Entity(1595): pocketmine\Player->checkBlockCollision()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #2 src/pocketmine/Player(1502): pocketmine\entity\Entity->move(double -0.045899999999989, double 0.012, double -0.25740000000002)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #3 src/pocketmine/Player(1662): pocketmine\Player->processMovement(integer 1)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #4 src/pocketmine/level/Level(751): pocketmine\Player->onUpdate(integer 151974)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #5 src/pocketmine/Server(2377): pocketmine\level\Level->doTick(integer 151974)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #6 src/pocketmine/Server(2534): pocketmine\Server->checkTickUpdates(integer 151974, double 1513790412.0609)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #7 src/pocketmine/Server(2274): pocketmine\Server->tick()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #8 src/pocketmine/Server(2149): pocketmine\Server->tickProcessor()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #9 src/pocketmine/Server(1737): pocketmine\Server->start()
2017-12-20 [18:20:12] [Server thread/DEBUG]: #10 src/pocketmine/PocketMine(554): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g3.hmserv.eu-19144/, string /home/gsp_1485/g3.hmserv.eu-19144/plugins/)
2017-12-20 [18:20:12] [Server thread/DEBUG]: #11 /home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar/src/pocketmine/PocketMine.php)```

# Paste your crashdump here (if theres any)
```

```text
# Paste the contents of your TeaSpoonDump here ( by running /bugreport ) [can be found at <TeaSpoon Plugin Folder>/dumps/<time>]
```

**Comment by NycuRO** [comment]
@CortexPE Seriosly?

**Comment by CortexPE** [comment]
u didnt fix #87 tho

**Comment by NycuRO** [comment]
oh lol
